app/views/users/show.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is the profile for a given a user</h1>
\ No newline at end of file
+<div class="center flex-column mt-3 mx-3">
+    <%= render partial: "shared/user", locals: {user: @user}%>
```

Comment:
- [ ] When I click on the user link, it shows this error:

<img width="683" alt="image" src="https://user-images.githubusercontent.com/77190203/173190739-e2dca678-b6c6-4bdd-a628-b0bf506ad3a2.png">

Please double-check that the correct path is displayed.